Dfe.PrepareConversions/Dfe.PrepareConversions/Pages/UpdateAcademyConversionProjectPageModel.cs

```diff
@@ -91,9 +91,14 @@ private void RePopDatePickerModelsAfterValidationFail()
       Project.LocalAuthorityInformationTemplateSentDate = AcademyConversionProject.LocalAuthorityInformationTemplateSentDate;
       Project.LocalAuthorityInformationTemplateReturnedDate = AcademyConversionProject.LocalAuthorityInformationTemplateReturnedDate;
    }
-   public static decimal? CalculateGrantAmount(string type, string phase)
+   public static decimal? CalculateGrantAmount(string type, string phase, string numberOfSites = "1")
    {
-      if (phase is null) return 25000;
+      int defaultAmount = 25000;
+      int supplementAmount = CalculateGrantSupplement(numberOfSites);
+
+      defaultAmount += supplementAmount;
+
+      if (phase is null) return defaultAmount;
```

Comment:
@dneed-nimble @paullocknimble 
Please could someone help me confirm whether I have understood correctly the intended behaviour 

https://dfe-gov-uk.visualstudio.com/Academies-and-Free-Schools-SIP/_workitems/edit/147236